### PR TITLE
HDDS-9409. Update SCM decommissioning documentation.

### PR DESCRIPTION
**Pull request by nandakumar131**
## What changes were proposed in this pull request?

SCM Decommissioning doc is updated with a note to delete the private key as part of decommissioning. This has to be done until we have certificate revocation support.

## What is the link to the Apache JIRA
[HDDS-9409](https://issues.apache.org/jira/browse/HDDS-9409)

## How was this patch tested?
Doc change. This doesn't need any testing.

### CODE REVIEW

File: hadoop-hdds/docs/content/feature/Decommission.md [hunk 1]
```diff
@@ -77,15 +77,16 @@ Storage Container Manager (SCM) decommissioning is the process in which you can
...
+ozone admin scm decommission [-hV] [-id=<scmServiceId>] [--scm=<scm>] -nodeid=<nodeId>
 ```
 You can obtain the 'nodeId' by executing this command, **"ozone admin scm roles"**
 
-**Note -** If you want to decommission a **primordial** scm, you may first transfer the leadership to a non primordial scm and then decommission the primordial scm.
+**Note -** If you want to decommission the **leader** scm, you must first transfer the leadership to a different scm and then decommission the node.
 
 To transfer the leader, we can excute below command,
 ```shell
 ozone admin scm transfer --service-id=scmservice -n ${nodeId}
```
Comment (LZD-PratyushBhatt) [inline hadoop-hdds/docs/content/feature/Decommission.md:88]: Should we write _scmservice_ also in some placeholder as this is not fixed right?

Comment (nandakumar131) [reply hadoop-hdds/docs/content/feature/Decommission.md]: Done!

File: hadoop-hdds/docs/content/feature/Decommission.md [hunk 2]
```diff
@@ -78,14 +78,15 @@ Storage Container Manager (SCM) decommissioning is the process in which you can
...
 ```
 You can obtain the 'nodeId' by executing this command, **"ozone admin scm roles"**
 
-**Note -** If you want to decommission a **primordial** scm, you may first transfer the leadership to a non primordial scm and then decommission the primordial scm.
+**Note -** If you want to decommission the **leader** scm, you must first transfer the leadership to a different scm and then decommission the node.
 
 To transfer the leader, we can excute below command,
 ```shell
-ozone admin scm transfer --service-id=scmservice -n ${nodeId}
+ozone admin scm transfer [-id=<scmServiceId>] -n=<nodeId>
```
Comment (adoroszlai) [inline hadoop-hdds/docs/content/feature/Decommission.md:88]: Please do not change `--service-id` to `-id`.  Both are supported by the command, but `-id`, as a multi-char short option, should really be deprecated (see description of HDDS-6736).

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @nandakumar131 for the patch, @LZD-PratyushBhatt for the review.

**Pull request merged**